swapfactory/swap_factory_test.go

```diff
@@ -44,7 +44,7 @@ func TestSwapFactory_NewSwap(t *testing.T) {
 
 	nonce := big.NewInt(0)
 	tx, err = contract.NewSwap(auth, [32]byte{}, [32]byte{},
-		ethcommon.Address{}, defaultTimeoutDuration, nonce)
+		ethcommon.Address{}, defaultTimeoutDuration, ethcommon.HexToAddress("0000000000000000000000000000000000000000"), big.NewInt(0), nonce)
```

Comment:
```suggestion
		ethcommon.Address{}, defaultTimeoutDuration, types.EthAssetETH, big.NewInt(0), nonce)
```

same for all others in this file?